### ISSUE TITLE
Ability to read memos

### ISSUE THREAD

**Issue by garethtdavies** [adityapk00/zcash-qt-wallet#13]
With the newly formatted transactions tab 👍 given you can now see the incoming transactions to shielded addresses the ability to view a memo using the context menu would be helpful i.e. in addition to "Copy txid" and "View on block explorer" a new item called "View memo".

**Comment by adityapk00** [comment]
Commit: 4bd43bf030e4100967020c59a130ba837b7e12a2

**Comment by garethtdavies** [comment]
Tested this and works nicely.

**Comment by adityapk00** [comment]
Add icon on transaction table row to indicate there is a memo here to be seen.

**Comment by adityapk00** [comment]
Added indicator in tx table that there is a memo. 

Commit: cf7b2e1c4b7ccdc13aa8aff6b0e00d64d121ce66

**Comment by garethtdavies** [comment]
I was expecting a little memo indicator (like a mail icon or something), should I be seeing S/R/M like this?

![image](https://user-images.githubusercontent.com/135382/47589474-163ad200-d91e-11e8-9db0-9e92bd716ab1.png)

**Comment by adityapk00** [comment]
Yeah, sorry. I just added "S" "R" for send/receive and "M" for memo, until I get some time to design icons for these :)

**Comment by garethtdavies** [comment]
Ah, I couldn't fathom what R means but yeh, send/receive/memo makes perfect sense. Working great then!

**Comment by adityapk00** [comment]
OK, added a default icon for now.

![image](https://user-images.githubusercontent.com/31996805/47590053-f1dff500-d91f-11e8-9f64-95a66bd9278e.png)
